packages/avatars/stories/examples/Default.tsx

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import { Story } from '@storybook/react';
+import { LG } from '@zendeskgarden/react-typography';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import UserIcon from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+import ZendeskIcon from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+const StyledTable = styled.table.attrs<Partial<IStoryProps>>(p => ({
+  style: { backgroundColor: p.surfaceColor }
+}))<Partial<IStoryProps>>`
+  width: 100%;
+  border-spacing: 0;
+`;
+
+const StyledAvatarTd = styled.td`
+  padding: ${p => p.theme.space.xs} 0;
```

Comment:
Might feel better OOTB with large/xl padding.

Reply:
Update to include large padding.